tuna/miopen/driver/convolution.py

```diff
@@ -191,43 +206,43 @@ def set_defaults(self, defaults):
                          k, self.spatial_dim)
 
   @staticmethod
-  def get_params(tok1):
+  def get_params(tok1: str) -> Callable[[str, str], bool]:
```

Comment:
This does not return a callable function. It attempts to return the value returned by get_fd_name. However get_fd_name returns nothing.
Remove this return (leave the function call) and set the return value to none.

Reply:
get_fd_name retruns tok of type str